src/extension.ts

```diff
@@ -100,40 +23,60 @@ export async function activate(context: vscode.ExtensionContext) {
   }
 }
 
-function getAllowedDomains(): string[] {
-  // Attempt to read the list of allowed domains
-  // from the central policy file (if it exists).
-  const filePath = path.join(os.homedir(), POLICY_FILE);
-  if (fs.existsSync(filePath)) {
-    const contents = fs.readFileSync(filePath, "utf8");
+class TestPolicyProvider implements vsls.PolicyProvider {
```

Comment:
Maybe `GatekeeperPolicyProvider`? We actually have customers using this, so I wouldn't want the code to somehow indicate that this is just a sample. It's actually meant for product use.